build/org.eclipse.birt.releng.maven/BIRT-Runtime-Minimized.aggr

```diff
@@ -40,6 +41,7 @@
   <mavenMappings namePattern="uk\.co\.spudsoft\.birt\.emitters\.excel" groupId="\$maven-groupId\$" artifactId="\$maven-artifactId\$" versionPattern="([^.]+)\.([^.]+)\.([^.]+)(\..*)?" versionTemplate="$1.$2.$3"/>
   <mavenMappings namePattern=".*" groupId="\$maven-groupId\$" artifactId="\$maven-artifactId\$" versionPattern=".*" versionTemplate="\$maven-version\$"/>
   <mavenMappings namePattern=".*" groupId="\$maven-wrapped-groupId\$" artifactId="\$maven-wrapped-artifactId\$" versionPattern=".*" versionTemplate="\$maven-wrapped-version\$"/>
+  <mavenDependencyMapping namespacePattern="osgi\.bundle" namePattern="org\.apache\.derby" groupId="*" artifactId="*" versionRangePattern=".*" versionRangeTemplate="major.minor.micro.1"/>
```

Comment:
Hard coding this to `.1` is eventually going to be wrong, so this is just a problem waiting to happen.  I believe it's currently mapping to a version range.   Is that the "problem" you are trying to address?  (Is it a problem?)  Maybe `.qualifier` would be more correct if you are trying to avoid a range.